app/models/mail_snippet.rb

```diff
@@ -50,4 +50,21 @@ def update_organization
     # Manually update to ensure that it runs the before save stuff
     organization && organization.update_attributes(updated_at: Time.current)
   end
+
+  private
+
+  def geocode_data
+    address
+  end
+
+  def geocode_columns
+    %i[address]
+  end
+
+  def should_be_geocoded?
```

Comment:
Not sure if this is an improvement, also not sure if it's possible, but can you use `super` here to use the concerns `should_be_geocoded?` ?

Reply:
> not sure if it's possible, but can you use super here to use the concerns should_be_geocoded?

Negative. With concerns, the model methods overwrite rather than override — there's no inheritance to leverage.